docs/administrators/operational_procedures.rst

```diff
@@ -15,6 +15,11 @@ Project Release for Production
 
 This process describes the steps to prepare an inmanta project for production release.
 
+All packages should be hosted in an internal, curated python repository (like nexus or devpi). For security reasons,
```

Comment:
I would add a qualifier that this does not apply for projects that only use public modules and Python dependencies.

Reply:
Added "All **private** packages (...)"

Reply:
That's not right.
- Projects that use only public modules and Python dependencies may safely use pypi. This is not clear from this paragraph.
- Projects with at least one private depedency should host ALL (not only private) packages in an internal, curated Python repository.

Reply:
> This is not clear from this paragraph.
Reading it again I see that it is there but it reads difficult to me: you say "you should not do x. You may use x if y". Why not just "unless y, you should not do x" or "you should not do x, except if y"?

Reply:
This is still a bit of an issue. You start with a statement saying "do x". Then you elaborate on how to "do x". And only then do you say that you may do it differently under certain conditions. The overall meaning isn't all that clear I think.

Reply:
Rephrased it in a clearer way

Reply:
Big improvement imo.